### DISCUSSION TITLE
DNS-over-HTTPS from iPhone shown as originating from router address

### DISCUSSION THREAD

**Discussion by mtheuma** [AdguardTeam/AdGuardHome#3514] [Q&A] [answered]
Hi,

I just setup DNS-over-HTTPS and I noticed that all requests from my iPhone are shown as originating from my router IP address when connected via WiFi. By contrast, my Windows PC shows the correct origin IP address for requests.

When using my iPhone over cellular, it does show the correct Client ID that I specified for the '.mobileconfig'.

Is this expected behaviour? Or did I configure something incorrectly?

Edit: I should also mention that I am not using Adguard Home as a DHCP server.

Thanks!

**Comment by ainar-g** [comment, marked as answer]
Hello, are you using v0.106.3?  The upcoming v0.107.0 release will have a fix for this through the `trusted_proxies` mechanism.  See #2799.  The latest beta should be out later today.

**Reply by mtheuma** [reply]
Yes, that must be it. Thanks!
